File: core/src/main/java/org/mifos/mobilewallet/core/domain/model/NewClient.java
package org.mifos.mobilewallet.core.domain.model;

/**
 * Created by naman on 20/8/17.
 */

public class NewClient {

    private String firstname;
    private String lastname;
    private String externalId;

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public String getExternalId() {
        return externalId;
    }

    public void setExternalId(String externalId) {
        this.externalId = externalId;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }
}

File: core/model/src/commonMain/kotlin/org/mifospay/core/model/client/NewClient.kt
/*
 * Copyright 2024 Mifos Initiative
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 *
 * See See https://github.com/openMF/kmp-project-template/blob/main/LICENSE
 */
package org.mifospay.core.model.client

import org.mifospay.core.common.Parcelable
import org.mifospay.core.common.Parcelize

@Parcelize
data class NewClient(
    val firstname: String,
    val lastname: String,
    val externalId: String,
    val mobileNo: String,
    val address: ClientAddress,
    val savingsProductId: Int,
) : Parcelable
